web-marketplace/src/pages/ProjectReview/ProjectReview.tsx

```diff
@@ -283,7 +285,10 @@ export const ProjectReview: React.FC<React.PropsWithChildren<unknown>> = () => {
               overflowY: 'scroll',
             })}
           >
-            <pre>{!!metadata && JSON.stringify(metadata, null, 2)}</pre>
+            <pre>
+              {!!metadata &&
+                JSON.stringify(omit(metadata, OMITTED_METADATA_KEYS), null, 2)}
```

Comment:
@erikalogie On the project review page, in the metadata section, should we display the whole anchored metadata? or exclude the fields such as name, location, etc. that are already displayed above. I've updated it to use the latter, let me know what you think, thanks.

Reply:
not sure if that's worth working on this further since we might upgrade this all together anyway

Reply:
Yeah I think is fine for now